lib/helper/WebDriverIO.js

```diff
@@ -1396,6 +1569,36 @@ class WebDriverIO extends Helper {
   }
 
   /**
+   * Waits for the specified value to be in value attribute
+   *
+   * ```js
+   * I.waitForValue("GoodValue", '//input');
+   * ```
+   */
+  waitForValue(value, context, sec = null) {
```

Comment:
this is inconsistent with `seeInField` or `fillField`. First is locator, second is value. Because locator is required.